### PR TITLE
Update save for later content and remove landing page

### PR DESCRIPTION
**Pull request by richtier**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] Change is accessible to the standards we subscribe to.
 - [x] Add a printscreen to the PR

image

### CODE REVIEW

File: core/templates/core/invalid-save-for-later-key.html [hunk 1]
```diff
@@ -5,17 +5,20 @@
...
       <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
-        <h2 class="govuk-error-summary__title" id="error-summary-title">
-          There is a problem
-        </h2>
+        <h2 class="govuk-error-summary__title" id="error-summary-title">Your link has expired</h2>
         <div class="govuk-error-summary__body">
           <ul class="govuk-list govuk-error-summary__list">
             <li>
-              <span>The saved form responses could not be found.</span>
+              <span>We can only save for your progress for 72 hours.</span>
```
Comment (eadpearce) [inline core/templates/core/invalid-save-for-later-key.html]: does this need to be in a `span`?

File: core/templates/core/invalid-save-for-later-key.html [hunk 2]
```diff
@@ -5,17 +5,20 @@
   <div class="govuk-grid-row govuk-!-margin-top-9">
     <div class="govuk-grid-column-two-thirds">
       <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
-        <h2 class="govuk-error-summary__title" id="error-summary-title">
-          There is a problem
-        </h2>
+        <h2 class="govuk-error-summary__title" id="error-summary-title">Your link has expired</h2>
```
Comment (eadpearce) [inline core/templates/core/invalid-save-for-later-key.html]: should this be an `h1`?

File: core/views.py [hunk 3]
```diff
@@ -44,26 +44,8 @@ def get_form(self, step=None, *args, **kwargs):
...
-            context.update({
-                'service_availability_start_date': settings.SERVICE_AVAILABILITY_START_DATE,
-                'service_availability_end_date': settings.SERVICE_AVAILABILITY_END_DATE,
-            })
-        return context
-
-
 class PrivacyPolicyView(TemplateView):
-    template_name = 'core/privacy-policy.html'
+    template_name = 'core/invalid-save-for-later-key.html'
```
Comment (eadpearce) [inline core/views.py]: is this correct? seems not to match with the view class name

File: core/templates/core/footer.html [hunk 4]
```diff
@@ -19,6 +19,11 @@ <h2 class="govuk-visually-hidden">Support links</h2>
               Accessibility statement
             </a>
           </li>
+          <li class="govuk-footer__inline-list-item">
+            <a class="govuk-footer__link" href="http://www.gov.uk/done/give-feedback-temporary-tariff-rates">
```
Comment (kowalcj0) [inline core/templates/core/footer.html]:
```suggestion
            <a class="govuk-footer__link" href="https://www.gov.uk/done/give-feedback-temporary-tariff-rates">
```

File: core/templates/core/invalid-save-for-later-key.html (hunk 1 above)
Comment (richtier) [reply core/templates/core/invalid-save-for-later-key.html]: it's copy and pasted from the design system :)

File: conf/urls.py [hunk 5]
```diff
@@ -13,8 +15,8 @@
 urlpatterns = [
     url(
         r'^$',
-        skip_ga360(core.views.LandingPageView.as_view()),
-        name='landing-page'
+        RedirectView.as_view(url=reverse_lazy('user-type-routing', kwargs={'step': constants.STEP_USER_TYPE})),
```
Comment (sdonk) [inline conf/urls.py]: https://docs.djangoproject.com/en/2.2/ref/class-based-views/base/#django.views.generic.base.RedirectView.pattern_name would that help?

File: core/views.py (hunk 3 above)
Comment (richtier) [reply core/views.py]: oops did it for debugging. fixed :)

File: conf/urls.py (hunk 5 above)
Comment (richtier) [reply conf/urls.py]: unfortunately not when the pattern includes kwargs :)

File: core/templates/core/footer.html (hunk 4 above)
Comment (kowalcj0) [reply core/templates/core/footer.html]: btw. https://www.gov.uk/done/give-feedback-temporary-tariff-rates returns `404`. Is that OK?

File: core/templates/core/invalid-save-for-later-key.html (hunk 1 above)
Comment (eadpearce) [reply core/templates/core/invalid-save-for-later-key.html]: haha well then my gripe is with design system

File: core/templates/core/footer.html (hunk 4 above)
Comment (richtier) [reply core/templates/core/footer.html]: expected as they're not turning that on until no deal happens :)

**Pull request merged**